Frends.PowerShell.RunScript/Frends.PowerShell.RunScript/PowerShell.cs

```diff
@@ -75,7 +78,7 @@ internal static SessionWrapper CreateSession()
         /// Executes a PowerShell script from a file or the script parameter
         /// </summary>
         /// <returns>Object { Result: List&lt;dynamic&gt;, Errors: List&lt;string&gt;, Log: string}</returns>
-        public static PowerShellResult RunScript(RunScriptInput input, [Browsable(false)]RunOptions options)
+        public static PowerShellResult RunScript(RunScriptInput input, [Browsable(false)] RunOptions options)
```

Comment:
Input and options should be in different tabs in Frends task editor, so add [Propertytab].